forge-gui/res/cardsfolder/upcoming/snowborn_simulacra.txt

```diff
@@ -0,0 +1,15 @@
+Name:Snowborn Simulacra
+ManaCost:X U U
+Types:Sorcery
+A:SP$ RepeatEach | ValidTgts$ Permanent.nonToken | TgtPrompt$ Select up to X target nontoken permanents | TargetMin$ X | TargetMax$ X | RepeatSubAbility$ DBConjure | RepeatTargeted$ True | SubAbility$ DBPutFromHand | StackDescription$ SpellDescription | SpellDescription$ Conjure a duplicate of each of X target nontoken permanents into your hand. Those cards perpetually gain "You may spend mana as though it were mana of any color to cast this spell." If X is 5 or more, you may put one of them from your hand onto the battlefield.
+SVar:DBConjure:DB$ MakeCard | DefinedName$ Remembered | Zone$ Hand | RememberMade$ True | ImprintMade$ True | SubAbility$ DBEffect | SubAbility$ DBCleanup
```

Comment:
DBCleanup on wrong line

But if you'd only create one Effect for all of them you wouldn't need it anyway...?

Reply:
Yeah, I'd remove both SubAbility$ params from this line (Forge will only do one anyway)

Here you don't want `RememberMade$` as that might interact weird with your RepeatEach

Then in line 4 do `SubAbility$ DBEffect` and in line 6 add `SubAbility$ DBPutFromHand`